source/aggregation-tutorials/one-to-one-join.txt

```diff
@@ -0,0 +1,208 @@
+
+The goal of this aggregation tutorial is to list all orders
+made in 2020, with the product information associated with each order.
+
+This example uses two collections:
+
+- ``products`` which contains documents detailing a shop's available products.
+- ``orders`` which contains documents describing an individual order for a product at that shop.
+
+Because an order can be for only one product, the aggregation uses a
+one-to-one join to match an order document to the document for the ordered
+product. The collections are joined using a field called ``product_id`` that exists in
+the documents in both collections.
+
+The examples and sample data in this tutorial are sourced from the `Practical MongoDB
+Aggregations Book <https://www.practical-mongodb-aggregations.com/>`__.
+
+Before You Get Started
+----------------------
+
+Before you get started with this tutorial, you will need a Node.js application
+with the {+driver-long+} installed that connects to a MongoDB deployment.
+
+To learn how to create a MongoDB deployment on MongoDB Atlas, or
+on a local MongoDB instance, see the :ref:`Connection Guide <node-connect-to-mongodb>`.
+
+To learn how to get started using MongoDB with a new Node.js project, see the
+:ref:`node-quickstart`.
+
+This tutorial uses sample data stored in two separate collections. Insert the
+following sample data into a collection called
+``products``:
+
+.. literalinclude:: /includes/aggregation/one-to-one-join.js
+   :language: javascript
+   :copyable: true
+   :start-after: // start-products
+   :end-before: // end-products
+   :dedent:
+
+Insert the following data into a collection called ``orders``:
+
+.. literalinclude:: /includes/aggregation/one-to-one-join.js
+   :language: javascript
+   :copyable: true
+   :start-after: // start-orders
+   :end-before: // end-orders
+   :dedent:
+
+For more information about inserting documents into a collection, see :ref:`node-fundamentals-insert-data`.
+
+One-to-One Join Tutorial
+------------------------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Build The Aggregation Pipeline
+
+      .. procedure::
+         :style: connected
+
+         .. step::
+
+            First, create a ``pipeline`` variable and add a :manual:`$match
+            </reference/operator/aggregation/match>` stage that matches orders made in 2020.
+
+            .. literalinclude:: /includes/aggregation/one-to-one-join.js
+               :language: javascript
+               :copyable: true
+               :start-after: // start-match
+               :end-before: // end-match
+               :dedent:
+
+         .. step::
+
+            The next stage in the pipeline is a :manual:`$lookup
+            </reference/operator/aggregation/lookup>` stage. The
+            ``$lookup`` stage joins the ``product_id`` field in the ``orders``
+            collection, to the ``id`` field in the ``products`` collection. The
+            following code adds a ``$lookup`` stage to the pipeline:
```

Comment:
I think we can delete for concision since this information is repeated. 
```suggestion
            collection, to the ``id`` field in the ``products`` collection.
```